### PR TITLE
IGNORE: Fail CI tasks immediately

### PR DESCRIPTION
**Pull request by sergiou87**
This is a test PR, :+1: :+1:  don't look! 🙈 🙈  :+1: :rocket: :zzz:

| Syntax      | Description |
| ----------- | ----------- |
| Header       | Title             |
| Paragraph  | Text  :+1:           |

 :astonished:

```
This will be code tag in a pre tag. -> What is here :astonished:
```
:no_mobile_phones:
<pre> just a pre tag :no_mobile_phones: tag </pre>

:computer:

` In some code, :computer:`

### CODE REVIEW

**Comment by Majkasgurunu** [comment]
#13624

**Review by serpa1 — COMMENTED** [review]
Testing notifications…

**Review by serpa1 — COMMENTED** [review]
Hello world! 👋

**Review by serpa1 — COMMENTED** [review]
Test comment:

<div class="TimelineItem TimelineItem--condensed pt-0 js-comment-container js-socket-channel js-updatable-content js-command-palette-pull-body" data-gid="PR_kwDOA32Mzs4tyHZ4" data-url="/desktop/desktop/pull/13201/partials/body" data-channel="eyJjIjoicHVsbF9yZXF1ZXN0Ojc2ODExMjI0OCIsInQiOjE2NDc1OTk3ODR9--13a63d6427d2ff2a0dae20347a5195a29318c6383cc805914c5c9767dd51ca60" style="box-sizing: border-box; padding: 4px 0px 0px; position: relative; display: flex; margin-left: 16px;"><div class="timeline-comment-group js-minimizable-comment-group js-targetable-element TimelineItem-body my-0" id="issue-1038242483" style="box-sizing: border-box; margin-top: 4px; margin-bottom: 0px !important; min-width: 0px; max-width: 100%; color: var(--color-fg-muted); flex: 1 1 auto;"><div id="pullrequest-768112248" class="timeline-comment-group js-minimizable-comment-group js-targetable-element my-0 comment previewable-edit js-task-list-container js-comment editable-comment timeline-comment--caret reorderable-task-lists timeline-comment ml-n3 unminimized-comment" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 0px !important; margin-left: -16px !important; position: relative; color: var(--color-fg-default); background-color: var(--color-canvas-default); border: 1px solid var(--color-border-default); border-radius: 6px;"><div style="box-sizing: border-box;"><div class="edit-comment-hide" style="box-sizing: border-box;"><task-lists disabled="" sortable="" style="box-sizing: border-box;"><div class="comment-body markdown-body js-comment-body soft-wrap user-select-contain d-block" style="box-sizing: border-box; display: block !important; font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; line-height: 1.5; overflow-wrap: break-word; width: 838px; padding: 16px; overflow: visible;"><p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px;">This is a test PR,<span> </span><g-emoji class="g-emoji" alias="+1" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">👍</g-emoji><span> </span><g-emoji class="g-emoji" alias="+1" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">👍</g-emoji><span> </span>don't look!<span> </span><g-emoji class="g-emoji" alias="see_no_evil" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f648.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">🙈</g-emoji><span> </span><g-emoji class="g-emoji" alias="see_no_evil" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f648.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">🙈</g-emoji><span> </span><g-emoji class="g-emoji" alias="+1" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">👍</g-emoji><span> </span><g-emoji class="g-emoji" alias="rocket" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f680.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">🚀</g-emoji><span> </span><g-emoji class="g-emoji" alias="zzz" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4a4.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">💤</g-emoji></p>

Syntax | Description
-- | --
Header | Title
Paragraph | Text 👍

<p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px;"><g-emoji class="g-emoji" alias="astonished" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f632.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">😲</g-emoji></p><div class="snippet-clipboard-content position-relative overflow-auto" style="box-sizing: border-box; position: relative !important; overflow: auto !important;"><pre style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Menlo, Consolas, &quot;Liberation Mono&quot;, monospace; font-size: 11.9px; margin-top: 0px; margin-bottom: 16px; overflow-wrap: normal; padding: 16px; overflow: auto; line-height: 1.45; background-color: var(--color-canvas-subtle); border-radius: 6px;"><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Menlo, Consolas, &quot;Liberation Mono&quot;, monospace; font-size: 11.9px; padding: 0px; margin: 0px; background: transparent; border-radius: 6px; word-break: normal; white-space: pre; border: 0px; display: inline; overflow: visible; line-height: inherit; overflow-wrap: normal;">This will be code tag in a pre tag. -&gt; What is here :astonished:
</code></pre></div><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px;"><g-emoji class="g-emoji" alias="no_mobile_phones" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4f5.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">📵</g-emoji></p><pre style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Menlo, Consolas, &quot;Liberation Mono&quot;, monospace; font-size: 11.9px; margin-top: 0px; margin-bottom: 16px; overflow-wrap: normal; padding: 16px; overflow: auto; line-height: 1.45; background-color: var(--color-canvas-subtle); border-radius: 6px;"> just a pre tag :no_mobile_phones: tag </pre><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px;"><g-emoji class="g-emoji" alias="computer" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4bb.png" style="box-sizing: border-box; display: inline-block; min-width: 1ch; font-family: &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;; font-size: 1em; font-weight: 400; line-height: 1; vertical-align: -0.075em; font-style: normal !important;">💻</g-emoji></p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 0px !important;"><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Menlo, Consolas, &quot;Liberation Mono&quot;, monospace; font-size: 11.9px; padding: 0.2em 0.4em; margin: 0px; background-color: var(--color-neutral-muted); border-radius: 6px;">In some code, :computer:</code></p></div></task-lists></div></div><form class="js-comment-update" id="issue-1038242483-edit-form" data-type="json" data-turbo="false" action="https://github.com/desktop/desktop/issues/13201" accept-charset="UTF-8" method="post" style="box-sizing: border-box;"></form><div class="pr-review-reactions" style="box-sizing: border-box;"><div data-view-component="true" class="comment-reactions js-reactions-container js-reaction-buttons-container social-reactions reactions-container flex-row-reverse flex-justify-end ml-3 d-flex" style="box-sizing: border-box; flex-direction: row-reverse !important; justify-content: flex-end !important; margin-left: 16px !important; display: none;"><form class="js-pick-reaction" data-turbo="false" action="https://github.com/desktop/desktop/reactions" accept-charset="UTF-8" method="post" style="box-sizing: border-box;"><div class="comment-reactions-options js-comment-reactions-options d-flex" style="box-sizing: border-box; display: flex !important;"></div></form></div></div></div></div></div><div id="js-timeline-progressive-loader" data-timeline-item-src="desktop/desktop/timeline_focused_item?after_cursor=Y3Vyc29yOnYyOpPPAAABf5yZhIgFqTkxNDE5Mzg5MA%3D%3D&amp;id=PR_kwDOA32Mzs4tyHZ4" style="box-sizing: border-box;"></div><div class="js-timeline-item js-timeline-progressive-focus-container" data-gid="CRE_kwDOA32Mzs5Eviyf" style="box-sizing: border-box;"><div class="TimelineItem" style="box-sizing: border-box; position: relative; display: flex; padding: 16px 0px; margin-left: 16px;"><div class="TimelineItem-badge" style="box-sizing: border-box; position: relative; z-index: 1; display: flex; width: 32px; height: 32px; margin-right: 8px; margin-left: -15px; color: var(--color-fg-muted); align-items: center; background-color: var(--color-timeline-badge-bg); border: 2px solid var(--color-canvas-default); border-radius: 50%; justify-content: center; flex-shrink: 0; font-family: -apple-system, &quot;system-ui&quot;, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-cross-reference"><path fill-rule="evenodd" d="M16 1.25v4.146a.25.25 0 01-.427.177L14.03 4.03l-3.75 3.75a.75.75 0 11-1.06-1.06l3.75-3.75-1.543-1.543A.25.25 0 0111.604 1h4.146a.25.25 0 01.25.25zM2.75 3.5a.25.25 0 00-.25.25v7.5c0 .138.112.25.25.25h2a.75.75 0 01.75.75v2.19l2.72-2.72a.75.75 0 01.53-.22h4.5a.25.25 0 00.25-.25v-2.5a.75.75 0 111.5 0v2.5A1.75 1.75 0 0113.25 13H9.06l-2.573 2.573A1.457 1.457 0 014 14.543V13H2.75A1.75 1.75 0 011 11.25v-7.5C1 2.784 1.784 2 2.75 2h5.5a.75.75 0 010 1.5h-5.5z"></path></svg></div><div class="TimelineItem-body" style="box-sizing: border-box; min-width: 0px; max-width: 100%; margin-top: 4px; color: var(--color-fg-muted); flex: 1 1 auto; font-family: -apple-system, &quot;system-ui&quot;, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><br class="Apple-interchange-newline">

**Review by serpa1 — COMMENTED** [review]
Another test

This is a test PR, :+1: :+1:  don't look! 🙈 🙈  :+1: :rocket: :zzz:

| Syntax      | Description |
| ----------- | ----------- |
| Header       | Title             |
| Paragraph  | Text  :+1:           |

 :astonished:

```
This will be code tag in a pre tag. -> What is here :astonished:
```
:no_mobile_phones:
<pre> just a pre tag :no_mobile_phones: tag </pre>

:computer:

` In some code, :computer:`

**Review by serpa1 — COMMENTED** [review]
This is a comment

**Review by sergiou87 — COMMENTED** [review]
test

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,6 +28,8 @@ jobs:
       # Needed for macOS arm64 until hosted macos-11.0 runners become available
       SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX11.1.sdk
     steps:
+      - name: Fail immediately # yes
```
Comment (sergiou87) [inline .github/workflows/ci.yml:28]: another test, now inline

**Comment by serpa1** [comment]
This is a normal comment

**Review by serpa1 — COMMENTED** [review]
This is a review comment

**Comment by serpa1** [comment]
Another normal comment

**Comment by Daddydyer** [comment]
Get off my shit before i find u

**Review by Gambinonull — COMMENTED** [review]
Hu

**Comment by KikeE36** [comment]
> Este es un PR de prueba, 👍 👍 ¡no mires! 🙈 🙈 👍 🚀 💤
> 
> Sintaxis	Descripción
> Encabezado	Título
> Párrafo	Mensaje de texto 👍
> 😲
> 
> ```
> This will be code tag in a pre tag. -> What is here :astonished:
> ```
> 
> 📵
> 
>  just a pre tag :no_mobile_phones: tag 
> 💻
> fail-ci-immediately
> ` In some code, :computer:`

**Comment by KikeE36** [comment]
[kikelagunaoyarbide](url)

**Pull request closed without merging**